spec/api-menu-spec.js

```diff
@@ -860,11 +860,15 @@ describe('Menu module', () => {
           submenu: [
             {
               label: 'Test Item',
-              accelerator: 'Ctrl+T',
-              click: () => {
-                // Test will succeed, only when the menu accelerator action
-                // is triggered
-                Promise.resolve()
+              accelerator: 'T',
+              click: (a, b, event) => {
+                expect(event).to.deep.equal({
+                  shiftKey: false,
+                  ctrlKey: false,
+                  altKey: false,
+                  metaKey: false,
+                  triggeredByAccelerator: true
```

Comment:
The docs say these object properties are all optional -- but we're testing for them here in the specs. Are they actually optional? If not, should we remove that from the docs?

Reply:
I'm not sure what optional would mean in this context. :/

From my testing all of these properties are available in the Event object upon triggering the MenuItem, so I figured that I'd test the integrity of the entire object (e.g. all key flags should be false since I registered the accelerator as `T`).

cc @MarshallOfSound since I believe he originally worked on this part of the code in this commit: https://github.com/electron/electron/commit/3b74837020b15a6c7ef74677f78bcab01a7466a9

P.S. Is it normal that https://electronjs.org/docs/api/structures/keyboard-event gives a 404?